### PR TITLE
Fix: Fixed issue where ftp addresses were not displayed in the path bar

### LINKED ISSUE #15031: Bug: Address bar is empty when opening an ftp server
### Description

image

### Steps To Reproduce

1. Open an ftp server.
2. See the address bar.

### Requirements

Show the ftp server address.

### Files Version

3.3.0.0

### Windows Version

10.0.22631.3296

### Log File

n/a

### PR DESCRIPTION
**Pull request by hishitetsu**
**Resolved / Related Issues**
- [X] Were these changes approved in an issue or discussion with the project maintainers? In order to prevent extra work, feature requests and changes to the codebase must be approved before the pull request will be reviewed. This prevents extra work for the contributors and maintainers.
   Closes #15031 

**Validation**
How did you test these changes?
- [X] Did you build the app and test your changes?
- [ ] Did you check for accessibility? You can use Accessibility Insights for this.
- [ ] Did you remove any strings from the en-us resource file?
   - [ ] Did you search the solution to see if the string is still being used? 
- [ ] Did you implement any design changes to an existing feature?
   - [ ] Was this change approved?
- [ ] Are there any other steps that were used to validate these changes?

### CODE REVIEW

File: src/Files.App/Utils/Storage/Helpers/StorageFileExtensions.cs [hunk 1]
```diff
@@ -297,7 +297,7 @@ private static PathBoxItem GetPathItem(string component, string path)
 			{
 				title = "SidebarNetworkDrives".GetLocalizedResource();
 			}
-			else if (component.Contains(':', StringComparison.Ordinal))
+			else if (component.EndsWith(":", StringComparison.Ordinal))
```
Comment (gumbarros) [inline src/Files.App/Utils/Storage/Helpers/StorageFileExtensions.cs:300]: I think it's better to use the `char` overload.

`component.EndsWith(':')`

Comment (hishitetsu) [reply src/Files.App/Utils/Storage/Helpers/StorageFileExtensions.cs:300]:
```suggestion
			else if (component.EndsWith(':'))
```

Comment (gumbarros) [reply src/Files.App/Utils/Storage/Helpers/StorageFileExtensions.cs]: Forgot about the `StringComparison.Ordinal` at my example 💀, you can keep it if you want, but I think it make no difference.

Comment (hishitetsu) [reply src/Files.App/Utils/Storage/Helpers/StorageFileExtensions.cs]: Comparison by char is done in `Ordinal`, so I've modified the code to be simpler as you said👍

**Pull request merged**